src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -106,25 +108,17 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
         }
 
         if (isVaultedInstrument(paymentData)) {
-            const paymentPayload = {
-                methodId,
-                paymentData: {
-                    formattedPayload: {
-                        bigpay_token: {
-                            token: paymentData.instrumentId,
-                        },
-                        verification_value: '000',
-                        credit_card_token: {
-                            token: JSON.stringify({
-                                checkoutId: this._digitalRiverCheckoutData.checkoutId,
-                            }),
-                        },
-                    },
-                    set_as_default_stored_instrument: shouldSetAsDefaultInstrument,
-                },
-            };
+            try {
+                return await this._submitVaultedInstrument(methodId, paymentData.instrumentId, this._digitalRiverCheckoutData.checkoutId, shouldSetAsDefaultInstrument, false);
+            } catch (error) {
+                if (!this._isAuthenticateSourceAction(error)) {
+                    throw error;
+                }
 
-            return await this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
+                const confirm: boolean = await this._authenticateSource(error.body.provider_data);
```

Comment:
No need for the type here:
```suggestion
                const confirm = await this._authenticateSource(error.body.provider_data);
```